libraries/AP_Baro/AP_Baro_Backend.h

```diff
@@ -23,6 +23,28 @@ class AP_Baro_Backend
     // If the value further that filtrer_range from mean value, it is rejected.
     bool pressure_ok(float press);
     uint32_t get_error_count() const { return _error_count; }
+
+    /*
+      device driver IDs. These are used to fill in the devtype field
+      of the device ID, which shows up as GND_BARO_ID* parameters to
+      users.
+     */
+    enum DevTypes {
```

Comment:
`enum class DevType`?

Reply:
this PR follows the same pattern as compass and INS

Reply:
We are mostly using "enum class" in new code though..